src/test/java/com/palantir/docker/compose/AggressiveShutdownStrategyIntegrationTest.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+ */
+
+package com.palantir.docker.compose;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+import com.palantir.docker.compose.configuration.ShutdownStrategy;
+import com.palantir.docker.compose.logging.DoNothingLogCollector;
+import org.junit.Test;
+
+public class AggressiveShutdownStrategyIntegrationTest {
+
+    @Test
+    public void shut_down_multiple_containers_immediately() throws Exception {
+        DockerComposeRule rule = DockerComposeRule.builder()
+                .file("src/test/resources/shutdown-strategy.yaml")
+                .logCollector(new DoNothingLogCollector())
+                .retryAttempts(0)
+                .shutdownStrategy(ShutdownStrategy.AGGRESSIVE)
+                .build();
+
+        assertThat(rule.dockerCompose().ps(), is(TestContainerNames.of()));
```

Comment:
Imagine you happened to have some containers running before your run this test. You'd get a failure in this test and dive into the code to start debugging.  You'd spend ages on this, until you eventually realise you just had a few containers up to start with and you'd just wasted hours of your life, so you add in one quick sanity check beforehand to save the next person ;)